prime-finder.js

```diff
@@ -1,3 +1,25 @@
 module.exports = function(n) {
-  // do work here
+  function isPrime(candidate) {
+    // loop up to sqrt of a number to determine whether it's prime
+    for (i = 2; i ** 2 <= candidate; i++) {
+      // if divisible by any numbers along the way, then not prime
+      if (candidate % i === 0 || candidate === 1) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  let primeCounter = 0;
+  let candidates = 2;
+  while (primeCounter < n) {
+    if (isPrime(candidates)) {
+      primeCounter++;
+      candidates++;
```

Comment:
Since `candidate++;` happens in both the if and the else, you can actually pull it out and just leave it after the if/else. Since there's nothing left in the else, the else just disappears. It should end up looking like this;
```
if (isPrime(candidates)) {
  primeCounter++;
}
candidates++;
```